WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Shipping Labels/ShippingLabelOrderItemsAggregatorTests.swift

```diff
@@ -0,0 +1,135 @@
+import XCTest
+@testable import WooCommerce
+import Yosemite
+
+final class ShippingLabelOrderItemsAggregatorTests: XCTestCase {
+    private let sampleSiteID: Int64 = 124
+    private let sampleOrderID: Int64 = 630
```

Comment:
⚠️ These are unused. 🙂 

```suggestion

```

Reply:
good catch! removed in https://github.com/woocommerce/woocommerce-ios/pull/3197/commits/b4cb7059e6f9f969691b3b8794c3229ae8ad32e8